test/cljc/reitit/openapi_test.clj

```diff
@@ -653,23 +652,22 @@
           (is (nil? (validate spec))))))))
 
 (deftest default-content-type-test
-  (doseq [[coercion ->schema]
-          [[#'malli/coercion (fn [nom] [:map [nom :string]])]
-           [#'schema/coercion (fn [nom] {nom s/Str})]
-           [#'spec/coercion (fn [nom] {nom string?})]]]
-    (testing coercion
+  (doseq [[coercion ->schema] [[malli/coercion (fn [nom] [:map [nom :string]])]
+                               [schema/coercion (fn [nom] {nom s/Str})]
+                               [spec/coercion (fn [nom] {nom string?})]]]
+    (testing (str coercion)
       (doseq [content-type ["application/json" "application/edn"]]
         (testing (str "default content type " content-type)
           (let [app (ring/ring-handler
                      (ring/router
                       [["/parameters"
                         {:post {:description "parameters"
-                                :coercion @coercion
+                                :coercion coercion
                                 :content-types [content-type] ;; TODO should this be under :openapi ?
-                                :parameters {:request {:content {"application/transit" (->schema :transit)}
-                                                       :body (->schema :default)}}
+                                :request {:content {"application/transit" {:schema (->schema :transit)}}
+                                          :body (->schema :default)}
```

Comment:
should this be under `:content :default `?

Reply:
added helpers to pull & tests to verify request & responses are read correctly